cromshell

```diff
@@ -43,12 +47,14 @@ fi
 
 # read env var, or use default server URL
 export CROMWELL_URL=${CROMWELL_URL:-"${CROMWELL_SERVER_FROM_FILE}"}
-FOLDER_URL=$( echo ${CROMWELL_URL} | sed -e 's#ht.*://##g' ) 
+FOLDER_URL=$( echo ${CROMWELL_URL} | sed -e 's#ht.*://##g' )
```

Comment:
I like the removal of the extra whitespace, but can you pull it out of this PR and into another?  It confuses the substantive changes here.

Reply:
Alternatively I have a syntax cleanup branch open that I can do this in (if you want to just revert the trailing whitespace lines).

Reply:
Ugh, this is IntelliJ's fault...

Reply:
So, when I pushed, IntelliJ automatically cleaned up the whitespace in the same commit as my changes, so there's no separate commit for the whitespace cleanup 😞